test/e2e/autoscale_test.go

```diff
@@ -119,9 +120,10 @@ func generateTraffic(
 		select {
 		case <-stopChan:
 			ctx.t.Log("Stopping generateTraffic")
-			if successfulRequests != totalRequests {
-				return fmt.Errorf("error making requests for scale up: total = %d, errors = %d, expected 0",
-					totalRequests, totalRequests-successfulRequests)
+			successRate := float32(successfulRequests)/float32(totalRequests)
```

Comment:
Define both total and successful at float64 and then avoid cast?

Reply:
> Also why float32? Why do you hate precision ;-)

Saving my precious stack space :rofl:.
I'll change to float64.

> Define both total and successful at float64 and then avoid cast?

Doing integer arithmetic with floating-point, what could go wrong?
Nothing in this case because the numbers are small, but still.

Reply:
Yeah here it should be fine, especially that we're just doing +1.